pkg/kapp/cmd/appchange/list.go

```diff
@@ -20,9 +22,12 @@ type ListOptions struct {
 	depsFactory cmdcore.DepsFactory
 	logger      logger.Logger
 
-	AppFlags cmdapp.Flags
+	AppFlags  cmdapp.Flags
+	TimeFlags TimeFlags
 }
 
+const dateFormat = "2006-01-02"
```

Comment:
nit: since its only being used in one place, lets inline

Reply:
Adopted.